### ISSUE TITLE
Running problem?

### ISSUE THREAD

**Issue by Angus1996** [xinario/SAGAN#9]
When I was runing the code，I met the following problem.Can you help me?
-----------------------------------------------------------------------------------------------------
DATA_ROOT=./datasets/experiment name=SAGAN which_direction=AtoB phase=test th test.lua
{
  input_nc : 3
  results_dir : "./results/"
  Size : 512
  batchSize : 1
  phase : "test"
  fineSize : 512
  aspect_ratio : 1
  how_many : "all"
  gpu : 1
  nThreads : 1
  DATA_ROOT : "./datasets/experiment"
  serial_batch_iter : 1
  preprocess : "regular"
  norm : "batch"
  which_epoch : "latest"
  name : "SAGAN"
  cudnn : 1
  serial_batches : 1
  flip : 0
  output_nc : 3
  which_direction : "AtoB"
  checkpoints_dir : "/home/angus/Documents/project/Medical_image/SAGAN/checkpoints"
  display_id : 200
  display : 0
}
Random Seed: 7774	
#threads...1	
Starting donkey with id: 1 seed: 7775
table: 0x41c20428
./datasets/experiment
trainCache	/home/angus/Documents/rfid-project/Medical_image/SAGAN/cache/_home_angus_Documents_project_Medical_image_SAGAN_datasets_experiment_test_trainCache.t7
Creating train metadata
serial batch:, 	1
table: 0x41dc8138
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
cmd..wc -L '/tmp/lua_RsZajh' |cut -f1 -d' '
64 samples found..... 0/64 .....................] ETA: 0ms | Step: 0ms          
Updating classList and imageClass appropriately
 [=================== 1/1 =====================>] Tot: 0ms | Step: 0ms          
Cleaning up temporary files
Dataset Size: 	64	
checkpoints_dir	/home/angus/Documents/project/Medical_image/SAGAN/checkpoints	
/home/angus/torch/install/bin/luajit: /home/angus/torch/install/share/lua/5.1/threads/threads.lua:315: /home/angus/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] /home/angus/torch/install/share/lua/5.1/dok/inline.lua:738: **<image.scale> could not find valid dest size**
stack traceback:
	[C]: in function 'error'
	/home/angus/torch/install/share/lua/5.1/dok/inline.lua:738: in function 'error'
	/home/angus/torch/install/share/lua/5.1/image/init.lua:718: in function 'scale'
	.../project/Medical_image/SAGAN/data/donkey_folder.lua:40: in function 'preprocessAandB'
	.../project/Medical_image/SAGAN/data/donkey_folder.lua:256: in function 'sampleHookTrain'
	...uments/project/Medical_image/SAGAN/data/dataset.lua:342: in function 'getByClass'
	...uments/project/Medical_image/SAGAN/data/dataset.lua:375: in function <...uments/project/Medical_image/SAGAN/data/dataset.lua:367>
	[C]: in function 'xpcall'
	/home/angus/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	/home/angus/torch/install/share/lua/5.1/threads/queue.lua:65: in function </home/angus/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	/home/angus/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:15: in main chunk

**Comment by xinario** [comment]
Do you have valid images inside "./datasets/experiment/test"?

**Comment by Angus1996** [comment]
> Do you have valid images inside "./datasets/experiment/test"?

Yes, I had done a preprocess. There are 906 images in the folder.
